PsyshockPhysics/Physics/Types/PhysicsComponentLookup.cs

```diff
@@ -92,6 +93,18 @@ public bool TryGetComponent(SafeEntity safeEntity, out T componentData)
             return new PhysicsComponentLookup<T> { lookup = componentDataFromEntity };
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Update(ref SystemState state)
+        {
+            lookup.Update(ref state);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Update(SystemBase system)
+        {
+            lookup.Update(system);
+        }
+
```

Comment:
It isn't quite obvious to me when you would want to use these over something like:
```csharp
processor.physicsCompLookup = SystemAPI.GetComponentLookup<MyComp>();
```
I do worry that having this API might confuse people, but I'll probably accept this change and apply it to PhysicsBufferLookup too.